index.html

```diff
@@ -18,6 +18,13 @@
       <div class="logo">
         <a class="logo-text" href="#">My Logo</a>
       </div>
+      <nav>
+        <ul class="desc-nav">
+          <li><a href="#">Portoflio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- [ ] As I can see here the `header` section is not aligned the same as the Figma design so according to the requirements please stick as much as possible with the Figma template 🙏🏻🙏🏻so kindly Fix the alignment here to be in the center with the background. and also Add the convenient and hight to the header section to match the design👌🏻👌🏻👌🏻

- Your design 
![header](https://user-images.githubusercontent.com/58553711/177781933-0bded839-596a-423f-99d0-4e2629dbd005.PNG)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177782078-247e8e97-0b9e-4de6-a3a9-cdae6c00e5da.png)